app/email/email.go

```diff
@@ -829,229 +802,6 @@ func (es *Service) SendLicenseUpForRenewalEmail(email, name, locale, siteURL, re
-
-	data.Props["Footer"] = T("api.templates.copyright")
-
-	body, err := es.templatesContainer.RenderToString("over_user_limit_workspace_suspended_body", data)
-	if err != nil {
-		return false, err
-	}
-
-	if err := es.sendMail(email, subject, body); err != nil {
-		return false, err
-	}
-
-	return true, nil
-}
-
-func (es *Service) SendOverUserFourteenDayWarningEmail(email string, locale string, siteURL string, overLimitDate string) (bool, error) {
-	T := i18n.GetUserTranslations(locale)
-
-	subject := T("api.templates.over_limit_14_days_subject")
-
-	data := es.NewEmailTemplateData(locale)
-	data.Props["SiteURL"] = siteURL
-	data.Props["Title"] = T("api.templates.over_limit_14_days_title")
-	data.Props["Info1"] = T("api.templates.over_limit_14_days_info1", map[string]interface{}{"OverLimitDate": overLimitDate})
-	data.Props["Button"] = T("api.templates.over_limit_fix_now")
-	data.Props["EmailUs"] = T("api.templates.email_us_anytime_at")
-
-	data.Props["Footer"] = T("api.templates.copyright")
-
-	body, err := es.templatesContainer.RenderToString("over_user_limit_7_days_body", data)
-	if err != nil {
-		return false, err
-	}
-
-	if err := es.sendMail(email, subject, body); err != nil {
-		return false, err
-	}
-
-	return true, nil
-}
-
-func (es *Service) SendOverUserSevenDayWarningEmail(email string, locale string, siteURL string) (bool, error) {
-	T := i18n.GetUserTranslations(locale)
-
-	subject := T("api.templates.over_limit_7_days_subject")
-
-	data := es.NewEmailTemplateData(locale)
-	data.Props["SiteURL"] = siteURL
-	data.Props["Title"] = T("api.templates.over_limit_7_days_title")
-	data.Props["Info1"] = T("api.templates.over_limit_7_days_info1")
-	data.Props["Button"] = T("api.templates.over_limit_fix_now")
-	data.Props["EmailUs"] = T("api.templates.email_us_anytime_at")
-
-	data.Props["Footer"] = T("api.templates.copyright")
-
-	body, err := es.templatesContainer.RenderToString("over_user_limit_7_days_body", data)
-	if err != nil {
-		return false, err
-	}
-
-	if err := es.sendMail(email, subject, body); err != nil {
-		return false, err
-	}
-
-	return true, nil
-}
-
-func (es *Service) SendSuspensionEmailToSupport(email string, installationID string, customerID string, subscriptionID string, siteURL string, userCount int64) (bool, error) {
-	// Localization not needed
-
-	subject := fmt.Sprintf("Cloud Installation %s Scheduled Suspension", installationID)
-	data := es.NewEmailTemplateData("en")
-	data.Props["CustomerID"] = customerID
-	data.Props["SiteURL"] = siteURL
-	data.Props["SubscriptionID"] = subscriptionID
-	data.Props["InstallationID"] = installationID
-	data.Props["SuspensionDate"] = time.Now().AddDate(0, 0, 61).Format("2006-01-02")
-	data.Props["UserCount"] = userCount
-
-	body, err := es.templatesContainer.RenderToString("over_user_limit_support_body", data)
-	if err != nil {
-		return false, err
-	}
-
-	if err := es.sendMail(email, subject, body); err != nil {
-		return false, err
-	}
-
-	return true, nil
-}
```

Comment:
So much awesome delete happening in this file :heart_eyes:

Reply:
I am using 1.18

Reply:
At least according to the ent issue, it worked for them if they downgraded. If you downgrade, does it work for you?

Reply:
I'll downgrade and then run the tool again